### PR TITLE
Create local_operated.json

### PR DESCRIPTION
**Pull request by ZEE6de**

### CODE REVIEW

**Review by natfoot — COMMENTED** [review]
What is being changed here?

**Review by natfoot — COMMENTED** [review]
Looks like a good edit.

**Comment by tyrasd** [comment]
this needs to be included in #179

File: data/fields/railway/local_operated.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "key": "railway:local_operated",
+    "type": "radio",
```
Comment (tyrasd) [inline data/fields/railway/local_operated.json:3]: should this be a `check` field, given that the values are only `yes` and `no`?

**Pull request closed without merging**